site/hugo.yaml

```diff
@@ -105,6 +105,9 @@ menu:
     - name: "0.9.0"
       url: "/in-dev/0.9.0/"
       parent: "releases"
+    - name: "1.0.0"
+      url: "/releases/1.0.0/"
```

Comment:
didn't fully get this part should the url be same as 0.9.0 i.e "/in-dev/0.9.0/"

Reply:
that should also needed to be fixed, but maybe in another PR.